test/scripts/e2e_go_tests.sh

```diff
@@ -89,18 +99,24 @@ if [[ "${ARCHTYPE}" = arm* ]]; then
     PARALLEL_FLAG="-p 1"
 fi
 
+PACKAGES="./..."
+if [ "$RUN_EXPECT" != "" ]; then
```

Comment:
I think this means that whether or not RUN_EXPECT is set to FALSE or TRUE (and it is set to FALSE by default at the top of this file) we will only ever run the expect tests.

Reply:
Opened #2863